internal/server/turn.go

```diff
@@ -54,20 +54,33 @@ func (s *SOMASServer) runOrgs() error {
 }
 
 // endOfTurn performs end of turn updates
+// TODO: organise order of end of turn actions
 func (s *SOMASServer) endOfTurn() error {
```

Comment:
Should we maybe inline this function into `runTurn`? We could also inline `runOrgs` there. 

I don't think the distinction between runOrgs/endOfTurn is really useful.